### DISCUSSION TITLE
播放yt-dlp下载的视频时，contextmenu_gui无法唤起菜单

### DISCUSSION THREAD

**Discussion by Ccchiiiho** [hooke007/mpv_PlayKit#309] [mpv 通用]
因为一直用的魔改的modern OSC，所以就需要一个右键菜单，MPV.NET_CM停更后就转投lazy的怀抱了，就用了contextmenu_gui这个脚本来实现右键菜单。之前一直用的好好的，但今天看自己用yt-dlp下载的视频时发现无法唤起右键菜单，打开命令行发现contextmenu_gui脚本报错显示attempt to concatenate a nil value，怀疑是视频里有什么参数让脚本报错了，以下是命令行截图
![image](https://github.com/hooke007/MPV_lazy/assets/70327841/36d12f17-1f69-4b7f-823c-8264265249c5)
这是yt-dlp的配置文件
![image](https://github.com/hooke007/MPV_lazy/assets/70327841/1950b34f-a6ea-4789-94e0-8f814eed3566)
其他视频暂时没有发现这个问题
![image](https://github.com/hooke007/MPV_lazy/assets/70327841/9b7c4038-aeac-46ab-94d5-1b65f81f9624)

**Comment by hooke007** [comment]
（可能）片源的问题，发链接给我看看

**Reply by Ccchiiiho** [reply]
我试过了，我用yt-dlp下载的所有文件都不行，yt-dlp是我自己下的版本，配置文件就上面这些

**Reply by hooke007** [reply]
<s>我的意思是，随便一个视频的样本链接，不是片源网址。</s>
不用了，我能复现了。

**Reply by Ccchiiiho** [reply]
本来还以为是默认包含的简介的问题，结果往配置文件里加了--no-write-info-json参数还是用不了，有什么解决方法吗

**Reply by hooke007** [reply]
一个快速修复 https://github.com/hooke007/MPV_lazy/commit/531a9fbb1a4540d472705b415f8b1a869387e5b8 看看能不能用

**Reply by Ccchiiiho** [reply]
能用了，感谢
